### PR TITLE
Adding Macbeth level to AutomatedTesting and converting to Prefab

### PR DESCRIPTION
**Pull request by SWMasterson**
No code changes, only assets added and material versions updated.

TestSuite_Main.py still passing:
.\python\python.cmd -m pytest --build-directory build_development\bin\profile  .\AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main.py
=== test session starts ===
platform win32 -- Python 3.7.10, pytest-5.3.2, py-1.9.0, pluggy-0.13.1
rootdir: F:\git\o3de, inifile: pytest.ini
plugins: mock-2.0.0, timeout-1.3.4, ly-test-tools-1.0.0
collected 26 items

AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main.py ........................... [100%]

=== 27 passed in 74.87s (0:01:14) ====

Signed-off-by: Sean Masterson <semaster@amazon.com>

### CODE REVIEW

File: Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/00_illuminant_tex.material [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+    "materialType": "../../Types/StandardPBR.materialtype",
+    "materialTypeVersion": 4,
     "properties": {
         "baseColor": {
-            "textureMap": "Materials/Presets/MacBeth/00_illuminant_sRGB.tif"
+            "textureMap": "00_illuminant_sRGB.tif"
         }
     }
-}
+}
```
Comment (smurly) [inline Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/00_illuminant_tex.material:11]: no new line at end of file. same with many other files

**Review by smurly — APPROVED** [review]
I'm not sure if the no new line at end of file is an issue or not but it's frequent throughout.

**Review by Adi-Amazon — APPROVED** [review]
I understand that all textures are hidden within the Pak file?

File: Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/00_illuminant_tex.material (hunk 1 above)
Comment (SWMasterson) [reply Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/00_illuminant_tex.material:11]: I didn't actually edit any of the files specifically. Everything was done by saving material files in the Material Editor.

**Comment by SWMasterson** [comment]
> 

This level.pak is a copy of the original from AtomTest - nothing changed here. The majority of the material files live in root\Gems\Atom\Feature\Common\Assets\Materials\Presets\MacBeth and then cached in the standardpbr.azmaterialtype

**Comment by SWMasterson** [comment]
> I understand that all textures are hidden within the Pak file?

I spoke with Mike Balfor and found there were files that could be removed. I have removed all unnecessary files from this checkin.

**Review by smurly — APPROVED** [review]

**Pull request merged**